config/crd/v1alpha1/networking.x-k8s.io_gateways.yaml

```diff
@@ -140,6 +140,7 @@ spec:
                         label. \n Support: Core"
                       maxLength: 253
                       minLength: 1
+                      pattern: ^(\*\.)?[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
```

Comment:
Can you revert this change? CI job is most probably failing due to this.

Reply:
Thanks a lot. Just fixed